test/todos-gateway.test.js

```diff
@@ -33,10 +36,18 @@ describe('todos-gateway', () => {
     await _todos.insertOne(_todo)
   })
 
-  describe('findAll()', () => {
+  describe('findTodos()', () => {
     it('should find and return a list of todos', async () => {
-      const todos = await findAll(_todos)
+      const todos = await findTodos(_todos)
       expect(todos[0]).to.deep.equal(_todo)
     })
   })
+
+  describe('insertTodo()', () => {
+    it('return the insertedTodo', async () => {
+      const data = { _id: uuid(), dueDate: '1/1/2018', 'task': 'Go for a jog.' }
+      const { ops } = await insertTodo(data)
+      expect(ops[0]).to.deep.equal(data)
```

Comment:
Notice how your test has to reach into the `ops`. Just make your function return what your really want.